src/main/java/com/bitso/Bitso.java

```diff
@@ -550,7 +589,8 @@ public BookOrder findMatchingOrders(String id) {
                     break outer;
                 }
             }
-            offset += limit; // increase the offset by the number of orders we just looked at
+            offset += limit; // increase the offset by the number of orders we
+                             // just looked at
```

Comment:
+Ya aplicando el <setting id="org.eclipse.jdt.core.formatter.insert_new_line_after_annotation_on_parameter" value="do not insert"/> con el hotkey Command + Shift + f sigue sucediendo.